### ISSUE TITLE
Export fails when using {exiftool:GROUP:TAG} in file/dir name template with TAG of type int

### ISSUE THREAD

**Issue by garz75** [RhetTbull/osxphotos#409]
**osxphotos version:** last (0.41.6)

**Steps to reproduce** : 
* use the export command with the --filename template, with an exiftool tag of type int (see below: example tag is {exiftool:EXIF:SubSecTimeOriginal}, present in any iPhone photo AFAIK)
* run command

**Expected result:** the export is successful

**Actual result:** osphotos crashes, see stack trace below.

It looks like a "cast" to String is needed. the values of the output of the ExifTool class may be integer, not always string...

My goal was trying to use a filename template that would "order" the burst photos I have in my library. I am glad you fixed the issues with the burst photos export...

```
BarneyStinson:~/Desktop/out/test> osxphotos export out --cleanup --update --report export_all_report-`date +%Y-%m-%d-%H%M%S`.txt --album "burst"  --convert-to-jpeg --jpeg-quality 0.95 --directory "{folder_album,NoAlbum}"  --person-keyword --keyword-template "{label}" --skip-original-if-edited --exiftool  --filename "{created.strftime,%Y-%m-%d-%H%M%S}-{exiftool:EXIF:SubSecTimeOriginal}" --jpeg-ext jpg --verbose
osxphotos version 0.41.6
exiftool path: /opt/homebrew/bin/exiftool
Using last opened Photos library: /Users/fgarzon/Pictures/Bibliothèque Photos.photoslibrary
Using export database /Users/fgarzon/Desktop/out/test/out/.osxphotos_export.db
Processing database /Users/fgarzon/Pictures/Bibliothèque Photos.photoslibrary/database/photos.db
Processing database /Users/fgarzon/Pictures/Bibliothèque Photos.photoslibrary/database/Photos.sqlite
Database locked, creating temporary copy.
Processing database.
Database version: 6000, 6.
Processing persons in photos.
Processing detected faces in photos.
Processing albums.
Processing keywords.
Processing photo details.
Processing import sessions.
Processing additional photo details.
Processing face details.
Processing photo labels.
Processing EXIF details.
Processing computed aesthetic scores.
Processing comments and likes for shared photos.
Done processing details from Photos library.
Exporting 1 photo to /Users/fgarzon/Desktop/out/test/out...
Traceback (most recent call last):
  File "/Users/fgarzon/.local/bin/osxphotos", line 8, in <module>
    sys.exit(cli())
  File "/Users/fgarzon/.local/pipx/venvs/osxphotos/lib/python3.9/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/Users/fgarzon/.local/pipx/venvs/osxphotos/lib/python3.9/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/Users/fgarzon/.local/pipx/venvs/osxphotos/lib/python3.9/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/fgarzon/.local/pipx/venvs/osxphotos/lib/python3.9/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/fgarzon/.local/pipx/venvs/osxphotos/lib/python3.9/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/Users/fgarzon/.local/pipx/venvs/osxphotos/lib/python3.9/site-packages/click/decorators.py", line 33, in new_func
    return f(get_current_context().obj, *args, **kwargs)
  File "/Users/fgarzon/.local/pipx/venvs/osxphotos/lib/python3.9/site-packages/click/decorators.py", line 21, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/Users/fgarzon/.local/pipx/venvs/osxphotos/lib/python3.9/site-packages/osxphotos/cli.py", line 1447, in export
    export_results = export_photo(
  File "/Users/fgarzon/.local/pipx/venvs/osxphotos/lib/python3.9/site-packages/osxphotos/cli.py", line 2461, in export_photo
    filenames = get_filenames_from_template(
  File "/Users/fgarzon/.local/pipx/venvs/osxphotos/lib/python3.9/site-packages/osxphotos/cli.py", line 2838, in get_filenames_from_template
    filenames, unmatched = photo.render_template(
  File "/Users/fgarzon/.local/pipx/venvs/osxphotos/lib/python3.9/site-packages/osxphotos/photoinfo/photoinfo.py", line 950, in render_template
    return template.render(
  File "/Users/fgarzon/.local/pipx/venvs/osxphotos/lib/python3.9/site-packages/osxphotos/phototemplate.py", line 283, in render
    return self._render_statement(
  File "/Users/fgarzon/.local/pipx/venvs/osxphotos/lib/python3.9/site-packages/osxphotos/phototemplate.py", line 308, in _render_statement
    results, unmatched = self._render_template_string(
  File "/Users/fgarzon/.local/pipx/venvs/osxphotos/lib/python3.9/site-packages/osxphotos/phototemplate.py", line 442, in _render_template_string
    vals = self.get_template_value_exiftool(
  File "/Users/fgarzon/.local/pipx/venvs/osxphotos/lib/python3.9/site-packages/osxphotos/phototemplate.py", line 924, in get_template_value_exiftool
    values = [sanitize_pathpart(value) for value in values]
  File "/Users/fgarzon/.local/pipx/venvs/osxphotos/lib/python3.9/site-packages/osxphotos/phototemplate.py", line 924, in <listcomp>
    values = [sanitize_pathpart(value) for value in values]
  File "/Users/fgarzon/.local/pipx/venvs/osxphotos/lib/python3.9/site-packages/osxphotos/path_utils.py", line 74, in sanitize_pathpart
    pathpart = pathpart.replace("/", replacement)
AttributeError: 'int' object has no attribute 'replace'
```

**Comment by RhetTbull** [comment]
I think your assessment is correct. I'm traveling and won't be able to get to this until the weekend but likely an easy fix. I'm surprised (and clearly didn't anticipate) this could happen as I'm reading the exiftool data from a string but I'll take a look.

**Comment by RhetTbull** [comment]
@garz75 I don't have access to my Mac right now so I can't push a release but I did make a fix directly in GitHub (first time doing that!) so if you want to clone the github repo and install that way, you can give it a try.   I'll push a release over the weekend.  Warning - the repo is large (> 1GB) due to the test files.

```sh
git clone https://github.com/RhetTbull/osxphotos.git
cd osxphotos
python3 setup.py install
```

**Comment by RhetTbull** [comment]
Fixed in release [v0.41.7](https://github.com/RhetTbull/osxphotos/releases/tag/v0.41.7)
